package.json

```diff
@@ -126,7 +127,8 @@
     "clean": "rimraf '{src,test,bin,standalone}/**/*.{js,map,d.ts}' 'package.zip' '.tmp' 'tmp'",
     "lint": "eslint . --ext .ts --config .eslintrc",
     "lint:fix": "npm run lint -- --fix",
-    "prebuild": "npm run clean && bash script/download-libs.sh",
+    "prebuild": "npm run clean",
+    "download:libs": "npm run clean && bash script/download-libs.sh",
```

Comment:
This name doesn't follow the rest of the naming patterns, where the colon is used for clarification or extensions. Maybe `download-libs`?

Reply:
yep makes sense to me chap